addins/Cake.Compression.yml

```diff
@@ -1,9 +1,8 @@
 Name: Cake.Compression
 NuGet: Cake.Compression
-Prerelease: true
 Assemblies:
 - "/**/Cake.Compression.dll"
-Repository: https://github.com/akordowski/Cake.Compression/
+Repository: https://github.com/akordowski/Cake.Compression
```

Comment:
@Jericho wondering if this is diff "enough" to warrant an update?

Reply:
We just need to decide what we want in our YAML files and I can make sure to standardize URLs. What do you think our standard should be: with or without slash?